boa/environment.py

```diff
@@ -329,5 +329,10 @@ def time_travel(
             assert blocks is not None  # mypy hint
             seconds = blocks * block_delta
 
-        self.evm.patch.timestamp += seconds
-        self.evm.patch.block_number += blocks
+        if (new_blknum := self.evm.patch.block_number + blocks) < 0:
```

Comment:
Do we really need to abbreviate?

Reply:
no :nerd_face:  but i think abbreviations are fine for variables with small scope! when i read code, i associate scope with size of the variable name, if the variable name is too long i start hunting for references to it outside of its actual scope.